manifests/init.pp

```diff
@@ -281,6 +290,7 @@
   String $file_default_mode                                     = $rundeck::params::file_default_mode,
   Boolean $security_roles_array_enabled                         = $rundeck::params::security_roles_array_enabled,
   Array $security_roles_array                                   = $rundeck::params::security_roles_array,
+  Hash $storage_encrypt_config                                  = $rundeck::params::storage_encrypt_config,
```

Comment:
Two little things here:
* We can improve this by enforcing the datatypes within the hash
* We can make it a bit more readable by setting the default directly in the rundeck class. since it's the same on all OSes, we don't need to define it in the params.pp (only useful if conditions are involved to determine the correct value)

```suggestion
  Hash[String,String] $storage_encrypt_config                   = {},
```

Reply:
Sure.

Thanks for the review, btw.